contracts/asset-proxy/contracts/src/libs/LibAssetData.sol

```diff
@@ -0,0 +1,420 @@
+/*
```

Comment:
This contract has been moved to the `contracts-dev-utils` package and can be deleted here.